### DISCUSSION TITLE
plays/fun-quiz

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#215] [Play Feedback]
# plays/fun-quiz

Its a Fun Quiz app which lets player to choose desirable category to answer 20 unique question with 4 options and pick the correct one.

https://www.reactplay.io/plays/fun-quiz

**Comment by atapas** [comment]
Remarkable Play, yay

**Reply by Angryman18** [reply]
it makes me feel so happy to see the effort i put is live now. thankq so much.

**Reply by atapas** [reply]
You did an awesome job in this play @Angryman18. Let the world know 🙌

Kudos to @koustov too for in-depth review.

**Reply by koustov** [reply]
❤  I just loved this play @Angryman18
